docs/zh/latest/plugins/openfunction.md

```diff
@@ -0,0 +1,179 @@
+---
+title: openfunction
+keywords:
+  - APISIX
+  - Plugin
+  - OpenFunction
+  - openfunction
+description: 本文介绍了关于 CNCF OpenFunction 插件的基本信息及使用方法。
+---
+
+<!--
+#
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+-->
+
+## 描述
+
+`openfunction` 插件用于将开源的分布式无服务器平台 [CNCF OpenFunction](https://openfunction.dev/) 作为动态上游集成至 APISIX。
+
+启用 `openfunction` 插件后，该插件会终止对已配置 URI 的请求，并代表客户端向 OpenFunction 的 function 发起一个新的请求，然后 `openfunction` 插件会将响应信息返回至客户端。
+
+## 属性
+
+| 名称                         | 类型    | 必选项 | 默认值  | 有效值       | 描述                                                         |
+| --------------------------- | ------- | ------ | ------- | ------------ | ------------------------------------------------------------ |
+| function_uri                | string  | 是     |         |              | OpenFunction function uri，例如 `https://localhost:30858/default/function-sample`。     |
+| ssl_verify                  | boolean | 否     | true    |              | 当设置为 `true` 时执行 SSL 验证。                            |
+| authorization               | object  | 否     |         |              | 访问 OpenFunction 的函数的授权凭证。|
+| authorization.service_token | string  | 否     |         |              | OpenFunction service token，其格式为 `xxx:xxx` ，支持函数入口的 basic auth 认证方式。 |
```

Comment:
```suggestion
| authorization.service_token | string  | 否     |         |              | OpenFunction service token，其格式为 `xxx:xxx`，支持函数入口的 basic auth 认证方式。 |
```

Reply:
> 支持函数入口的 basic auth 认证方式。

Can you give a reference link?

Reply:
This is the link of openfunction gateway which uses k8s gateway.
https://openfunction.dev/docs/concepts/networking/gateway/

The define of function entry of openfunction (https://openfunction.dev/docs/concepts/networking/function-entrypoints/) which use k8s gateway to visit function from outsider.

basic auth is  just a easy auth way that almost every k8s gateway has applied.